### ISSUE TITLE
Cause gap on Filament Sidebar

### ISSUE THREAD

**Issue by MarJose123** [stechstudio/filament-impersonate#54]
https://github.com/stechstudio/filament-impersonate/blob/ee92ce42de52561aaf6731390b92846329f58b13/resources/views/components/banner.blade.php#L37C4-L39

```css
    div.fi-layout > aside.fi-sidebar {
        padding-{{ $position }}: var(--impersonate-banner-height);
    }
```
This line of code cause an issues with the filament sidebar, it create a gap between the sidebar and the impersonate banner.

I'm using the default setup/configuration of the plugin.

![image](https://github.com/stechstudio/filament-impersonate/assets/18107626/1b573019-4928-4e9f-b52e-514b29ebead8)

---

Disabling the Css it fixed the issue or if the `position` setting is `top` then the css will not be applied.

![image](https://github.com/stechstudio/filament-impersonate/assets/18107626/916d6260-a1a7-4cfa-bee7-0330616f386f)
![image](https://github.com/stechstudio/filament-impersonate/assets/18107626/418b4a2f-7354-48b0-a820-dc9e4598b97e)

**Comment by edwink75** [comment]
I added this code to my `theme.css` file and that fixed the issue
```
div.fi-layout > aside.fi-sidebar {
    padding-top: 0px!important;
}
```

**Comment by MarJose123** [comment]
> I added this code to my `theme.css` file and that fixed the issue
> 
> ```
> div.fi-layout > aside.fi-sidebar {
>     padding-top: 0px!important;
> }
> ```

Yeah, we can have this as temporary fix on our `app.css`

**Comment by jvanloey** [comment]
Another option is publishing the views and removing these lines from the code.

**Comment by dennisoderwald** [comment]
The problem is superficially solved, but when scrolling in a larger table, it occurs again.

**Comment by V13Axel** [comment]
Should be fixed as of 3.6, just released
